azurerm/internal/services/web/tests/resource_arm_app_service_test.go

```diff
@@ -514,6 +514,27 @@ func TestAccAzureRMAppService_oneIpRestriction(t *testing.T) {
 	})
 }
 
+func TestAccAzureRMAppService_completeIpRestriction(t *testing.T) {
```

Comment:
Could we have an additional test that updates an app_service by adding, then removing, a 2nd `ip_restriction`? (or extend this one)